lib/rialto/etl/cli/extract.rb

```diff
@@ -16,7 +16,7 @@ def call(name)
         desc 'list', 'List callable extractors'
         # List callable extractors
         def list
-          callable_extractors = Rialto::Etl::Extractors.constants.reject { |name| name.to_s.include?('Abstract') }
+          callable_extractors = Rialto::Etl::Extractors.constants.map(&:to_s) - ['StanfordClient']
```

Comment:
Now that we don't need to do a substring check against the names of the extractors, I don't believe we need to coerce to a string here. How would you feel about changing this method to look like:

```ruby
callable_extractors = Rialto::Etl::Extractors.constants - [StanfordClient] # or .constants.reject { |name| name == StanfordClient }
say "Extractors supported: #{callable_extractors.join(', ')}"
```

:question:

Reply:
Alternatively, we could move the Stanford client out of the `extractors` dir into a new area for clients?

Reply:
I'm cool with that, too.